segmentation/include/pcl/segmentation/grabcut_segmentation.h

```diff
@@ -259,9 +259,8 @@ namespace pcl
         GaussianFitter (float epsilon = 0.0001)
           : sum_ (Eigen::Vector3f::Zero ())
```

Comment:
These are constructed in the field initializer list? Maybe set them explicitly to these values? Not sure what Eigen::Vector3f sum_{}; gives.